src/applications/vaos/tests/reducers/newAppointment.unit.spec.js

```diff
@@ -154,26 +154,27 @@ describe('VAOS reducer: newAppointment', () => {
       typeOfCareId: '323',
     };
 
-    it('should set systems when facility page is done loading', () => {
+    it('should set parentFacilities when facility page is done loading', () => {
       const currentState = {
         ...defaultState,
       };
       const action = {
         ...defaultOpenPageAction,
-        systems: systemsParsed,
+        parentFacilities: systemsParsed,
```

Comment:
Same comment as above: can we rename `systemsParsed` to `parentFacilitiesParsed`?